manim/mobject/geometry.py

```diff
@@ -1293,6 +1293,55 @@ def __init__(self, direction=RIGHT, buff=0, **kwargs):
             direction = np.append(np.array(direction), 0)
         Arrow.__init__(self, ORIGIN, direction, buff=buff, **kwargs)
 
+    def coordinate_label(self, integer_labels=True, n_dim=2, color=WHITE):
+        """A label based on the coordinates of the vector.
+
+        Parameters
+        ----------
+        integer_labels : :class:`bool`, optional
+            Whether or not to round the coordinates to integers.
+        n_dim : `class`:`int`, optional
+            The number of dimensions of the vector.
+        color : :class:`~.Colors`, optional
+            The color of the label.
```

Comment:
I'd rather have the types as actual type hints. Also, I'm pretty sure that WHITE is just a `str`.

```suggestion
    def coordinate_label(self, integer_labels: bool = True, n_dim : int = 2, color: str = WHITE):
        """A label based on the coordinates of the vector.

        Parameters
        ----------
        integer_labels
            Whether or not to round the coordinates to integers.
        n_dim
            The number of dimensions of the vector.
        color
            The color of the label.
```